sdk/python/foundation-models/benchmarking/evaluation_pipelines/claude-2_1/boolq.yaml

```diff
@@ -0,0 +1,217 @@
+      timeout: 900
+    inputs:
+      dataset_name: boolq
+      configuration: all
+      split: validation
+    outputs:
+      output_dataset:
+        type: uri_folder
+  sampler:
+    type: command
+    component: azureml://registries/azureml/components/dataset_sampler/labels/latest
+    limits:
+      timeout: 900
+    inputs:
+      dataset:
+        type: uri_folder
+        path: ${{parent.jobs.downloader.outputs.output_dataset}}
+      sampling_style: head
+      sampling_ratio: ${{parent.inputs.sample_ratio}}
+      random_seed: 0
+    outputs:
+      output_dataset:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  preprocessor:
+    type: command
+    component: azureml://registries/azureml/components/dataset_preprocessor/labels/latest
+    limits:
+      timeout: 900
+    inputs:
+      dataset:
+        type: uri_folder
+        path: ${{parent.jobs.sampler.outputs.output_dataset}}
+      template_input: "{\n  \"question\": {{question}},\n  \"answerA\": true,\n  \"\
+        answerB\": false,\n  \"context\": {{passage}},\n  \"target\": {{answer}}\n\
+        }"
+    outputs:
+      output_dataset:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  fewshot_downloader:
+    type: command
+    component: azureml://registries/azureml/components/dataset_downloader/labels/latest
+    limits:
+      timeout: 900
+    inputs:
+      dataset_name: boolq
+      configuration: all
+      split: train
+    outputs:
+      output_dataset:
+        type: uri_folder
+  fewshot_sampler:
+    type: command
+    component: azureml://registries/azureml/components/dataset_sampler/labels/latest
+    limits:
+      timeout: 900
+    inputs:
+      dataset:
+        type: uri_folder
+        path: ${{parent.jobs.fewshot_downloader.outputs.output_dataset}}
+      sampling_style: random
+      sampling_ratio: 1.0
+      random_seed: 0
+    outputs:
+      output_dataset:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  fewshot_preprocessor:
+    type: command
+    component: azureml://registries/azureml/components/dataset_preprocessor/labels/latest
+    limits:
+      timeout: 900
+    inputs:
+      dataset:
+        type: uri_folder
+        path: ${{parent.jobs.fewshot_sampler.outputs.output_dataset}}
+      template_input: "{\n  \"question\": {{question}},\n  \"answerA\": true,\n  \"\
+        answerB\": false,\n  \"context\": {{passage}},\n  \"target\": {{answer}}\n\
+        }"
+    outputs:
+      output_dataset:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  promptcrafter:
+    type: command
+    component: azureml://registries/azureml-preview-test1/components/prompt_crafter_with_few_shot_pattern/labels/latest
```

Comment:
Change to "azureml://registries/azureml/components/prompt_crafter/labels/latest" when the new version release is done

Reply:
Done